application/models/Question.php

```diff
@@ -1725,16 +1723,30 @@ public function getScaledSubquestions()
      */
     public function questionThemeNameValidator()
     {
-        // As long as there is a question theme name, and it's not 'core', it's ok.
+        /* need a type */
+        if (empty($this->type)) {
+            return null;
+        }
+        /* not needed in child question */
+        if (!empty($this->parent_qid)) {
+            return null;
+        }
         if (!empty($this->question_theme_name) && $this->question_theme_name != 'core') {
-            return;
+            $criteria = new CDbCriteria();
```

Comment:
Not sure if we should put this into a new method, just as to give some strength to the liason in between the models, and a have a more notorious place to update if needed.

Reply:
Maybe , but before must understand why create a broken solution in QuestionTheme : https://github.com/LimeSurvey/LimeSurvey/blob/6e5a5767219ee97ce8f81cfbc85c20b4cd4331c8/application/models/QuestionTheme.php#L1084


I can update here : https://github.com/LimeSurvey/LimeSurvey/blob/6e5a5767219ee97ce8f81cfbc85c20b4cd4331c8/application/models/QuestionTheme.php#L629 
If needed, but i think it's out of fix issue. It's rewriting for dev.

Reply:
> Maybe , but before must understand why create a broken solution in QuestionTheme :

That code is not recent, so I will try to recall.
I understand it was related to the situation when creating a question.
There is a point in time where it is not saved yet, but still need to use the same mechanisms as if it would be being edited. Then, the relationship with QuestionTheme is not materialized yet, then we instantiate a dummy question theme to be used temporarily.

Reply:
> Still, I was thinking just about a new static method `QuestionTheme::ByNameAndType`

Maybe it's better to use existing method ?